designs/black-parrot/descriptor.yaml

```diff
@@ -0,0 +1,432 @@
+origin:
+  - repository: https://github.com/black-parrot/black-parrot.git
+    revision: 45a28bf96e58f55687f8e09b2521ceada121ad95
+    licenses:
+      - LICENSE
```

Comment:
I think both the basejump_stl and HardFloat submodules used by the BP repo have different licenses and copyright holders from the BP repo itself, if that is indeed the case please add a separate entry and license files for them, so imported code can be properly attributed to its authors. See e.g. OpenPiton for reference: https://github.com/verilator/rtlmeter/blob/main/designs/OpenPiton/descriptor.yaml